data/filedefs/filetypes.Dockerfile.conf

```
@@ -0,0 +1,14 @@
+[styling=Sh]
+
+[keywords]
+primary=ADD ARG CMD COPY ENTRYPOINT ENV EXPOSE FROM HEALTHCHECK LABEL MAINTAINER ONBUILD RUN SHELL STOPSIGNAL USER VOLUME WORKDIR
+
+[lexer_properties=C]
```

```suggestion
[lexer_properties=Sh]
```
Or am I missing something?
It won't make much difference as the lexer properties for the C lexer will be probably ignored by the Sh lexer. But to be less confusing, I think it inherit from "Sh".